src/lib/api/user-sessions.ts

```diff
@@ -70,11 +70,13 @@ export const UserSessionApi = {
    *
    * This logs a user out of the system by deleting the active cookie session.
    *
-   * @param id The ID of the user session to delete
    * @returns Promise resolving to the deleted UserSession object
    */
-  delete: async (id: Id): Promise<UserSession> =>
-    EntityApi.deleteFn<null, UserSession>(`${USER_SESSIONS_BASEURL}/${id}`),
+  delete: async (_id: Id): Promise<UserSession> =>
+  // TODO we are not using the id here as the backend extracts the user's ID from the session cookie
+  // at some point it may make sense to decouple user session related network operations from
+  // entity related network operations.
+    EntityApi.deleteFn<null, UserSession>(`${USER_SESSIONS_LOGOUTURL}`),
```

Comment:
I decided to not attempt to refactor this so as to not bloat the PR. I picture extracting the Axios/SWR related lower level code into a namespace called `HTTPS` or something similar that can be shared between `EntityApi` and `AuthApi`. 
I consulted AI on this and the response I received was that it was common practice to separate CRUD-like operations on data from Auth operations for this exact type of reason. Open to thoughts, but either way I decided to defer for now.

Reply:
Seems reasonable Caleb and thanks for the thoughtfulness you put into this.